### DISCUSSION TITLE
Syntax highlighting doesnot work in  orgmode file

### DISCUSSION THREAD

**Discussion by Asrtodex** [NvChad/NvChad#1829] [General]
i installed org mode by using this line in plugin/init.lua
```
{
"nvim-orgmode/orgmode",
opt = function()
    return require "plugins.configs.orgmode"
end,
config = function(_,opts)
   require('orgmode').setup(opts)
end,
}``
```
and in the plugin/config/orgmode
added the default sugestion in github for ogmode ```

require('orgmode').setup_ts_grammar()

-- Treesitter configuration
require('nvim-treesitter.configs').setup {
  -- If TS highlights are not enabled at all, or disabled via `disable` prop,
  -- highlighting will fallback to default Vim syntax highlighting
  highlight = {
    enable = true,
    -- Required for spellcheck, some LaTex highlights and
    -- code block highlights that do not have ts grammar
    additional_vim_regex_highlighting = {'org'},
  },
  ensure_installed = {'org'}, -- Or run :TSUpdate org
}

require('orgmode').setup({
  org_agenda_files = {'~/Dropbox/org/*', '~/my-orgs/**/*'},
  org_default_notes_file = '~/Dropbox/org/refile.org',
})
```

and added 
{ name = 'orgmode' } 
to  plugin/configs/cmp

and other stuff like what org mode asking to

but the syntax hightlighting doesnot work

**Comment by siduck** [comment]
hi @Asrtodex dont use the opt function. we use it because we want it to be overridable by the user 
```lua
{
"nvim-orgmode/orgmode",
lazy=false,
config = function()
   local opts blabla
   require('orgmode').setup(opts)
end,
}``
```

**Reply by Asrtodex** [reply]
with lazy false there is so much error
so i skipp lazy to false
the org mode is fine but the color of the text is just white not using base color scheme that providing by nvchad

**Reply by Asrtodex** [reply]
thanks now it work the color scheme its work and great
